app/src/main/java/blockeq/com/stellarwallet/fragments/WalletFragment.kt

```diff
@@ -50,47 +60,116 @@ class WalletFragment : BaseFragment(), OnLoadAccount {
         endPollingAccount()
     }
 
+    //region User Interface
+
     private fun setupUI() {
+        bindAdapter()
         loadBalance()
     }
 
-    private fun loadBalance() {
+    private fun bindAdapter() {
+        recyclerViewArrayList = WalletHeterogenousArray(TotalBalance(loadBalance()),
```

Comment:
Nice that the data organization in the array is off loaded and not in the activity 👍